### ISSUE TITLE
Unable to click on Programs button

### ISSUE THREAD

**Issue by rakosi2** [emmVRC/emmVRC-Public#1625]
emmVRC: 3.1.1
If the file `VRChat\UserData\emmVRC\programs.json` exists then clicking on the Programs button will throw this error and doesn't open the submenu.

```
[17:02:30.378] [emmVRCLoader] [ERROR] Could not load the programs JSON file. Please make sure everything is correctly formatted!
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> emmVRC.TinyJSON.DecodeException: Variant is expected to be a ProxyArray here, but it is not.
  at emmVRC.TinyJSON.JSON.DecodeList[T] (emmVRC.TinyJSON.Variant data) [0x00019] in <0295954753964237b323db4e734c1071>:0
  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00032] in <e1319b7195c343e79b385cd3aa43f5dc>:0
   --- End of inner exception stack trace ---
  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00048] in <e1319b7195c343e79b385cd3aa43f5dc>:0
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <e1319b7195c343e79b385cd3aa43f5dc>:0
  at emmVRC.TinyJSON.JSON.DecodeType[T] (emmVRC.TinyJSON.Variant data) [0x001cd] in <0295954753964237b323db4e734c1071>:0
  at emmVRC.TinyJSON.JSON.MakeInto[T] (emmVRC.TinyJSON.Variant data, T& item) [0x00000] in <0295954753964237b323db4e734c1071>:0
  at emmVRC.TinyJSON.Variant.Make[T] () [0x00000] in <0295954753964237b323db4e734c1071>:0
  at emmVRC.Functions.Other.Programs.GetPrograms () [0x00060] in <0295954753964237b323db4e734c1071>:0
[17:02:30.379] [ERROR] Exception in IL2CPP-to-Managed trampoline, not passing it to il2cpp: System.NullReferenceException: Object reference not set to an instance of an object
  at emmVRC.Menus.ProgramsMenu.OpenMenu () [0x00084] in <0295954753964237b323db4e734c1071>:0
  at emmVRC.Menus.ProgramsMenu+<>c.<OnSceneWasLoaded>b__6_0 () [0x00000] in <0295954753964237b323db4e734c1071>:0
  at (wrapper dynamic-method) UnhollowerRuntimeLib.DelegateSupport.(il2cpp delegate trampoline) System.Void(intptr,UnhollowerBaseLib.Runtime.Il2CppMethodInfo*)
```

**Comment by thetrueyoshifan** [comment]
Can you send the program.json you're using?

**Comment by rakosi2** [comment]
a freshly generated program.json file
I can't upload jsons here so I sent it to you in Discord

**Comment by thetrueyoshifan** [comment]
Fixed with emmVRC v3.1.2
